components/wmtk_components/delaunay/internal/delaunay_geogram.cpp

```diff
@@ -49,7 +49,8 @@ std::tuple<Eigen::MatrixXd, Eigen::MatrixXi> delaunay_geogram(
 
     // Run!
     geo_assert(points.size() > 0);
-    engine->set_vertices(static_cast<GEO::index_t>(points.rows()), points.data());
+    Eigen::MatrixXd points_transposed = points.transpose();
```

Comment:
sorry about introducing this - but i do not think any unit tests failed with this. should we have a unit test that can detect this sort of thing? (i.e at least a bit of one example run whose values have to make sense)

Reply:
I created an issue for that.